### PR TITLE
added a date check as suggested in #202

### PR DESCRIPTION
**Pull request by nevrome**
Just a simple check we can easily do. As suggested in #202.

### CODE REVIEW

File: src/Poseidon/Janno.hs [hunk 1]
```diff
@@ -97,6 +98,30 @@ instance Show JannoSex where
...
+
+instance ToJSON BCADAge where
+    toEncoding = genericToEncoding defaultOptions
+
+instance FromJSON BCADAge
+
+instance Csv.FromField BCADAge where
+    parseField x = do
+        val <- Csv.parseField x
+        if val >= 2022 -- the current year
```
Comment (stschiff) [inline src/Poseidon/Janno.hs:114]: Haha, now we have a once-per-year maintenance task to do here :-) Better get the current year programmatically, right? I'll quickly look into it.

**Review by stschiff — COMMENTED** [review]
Nice! Definitely useful. Let's see whether we can get rid of the hard-coded 2022...

**Comment by stschiff** [comment]
https://stackoverflow.com/questions/52490035/how-to-get-date-in-haskell

File: src/Poseidon/Janno.hs (hunk 1 above)
Comment (nevrome) [reply src/Poseidon/Janno.hs:114]: Ah - right - that was stupid on my end :facepalm:

**Comment by nevrome** [comment]
Alright - thanks!

**Pull request merged**